pkg/util/utils.go

```diff
@@ -537,35 +537,6 @@ func OpenExclusiveFile(path string) (*os.File, error) {
 	return os.OpenFile(path, os.O_RDWR|os.O_CREATE|os.O_EXCL, 0666)
 }
 
-// PullType whether to pull new image
```

Comment:
This is a API Breaking change, we can not remove these but we can reroute them.

Reply:
Take a look at https://github.com/containers/storage/pull/703 to still define these constants and the validate function here, but have them use containers/common.

Reply:
fixed

Reply:
You just removed the change. I want you to change util.go to use containers/common so the code will be the same.